assets/js/pages/ClusterDetails/HanaClusterDetails.jsx

```diff
@@ -150,15 +153,27 @@ function HanaClusterDetails({
               },
               {
                 title: 'SID',
-                content: enrichedSapSystem,
-                render: (content) => (
-                  <SapSystemLink
-                    sapSystemId={content?.id}
-                    systemType="databases"
-                  >
-                    {content?.sid}
-                  </SapSystemLink>
-                ),
+                content: enrichedSapSystems,
+                render: (content) =>
+                  content.every(
+                    (sapSystem) => sapSystem.id && sapSystem.sid
+                  ) ? (
+                    <div>
+                      {content.map(({ id, sid: sapSystemSid }) => (
+                        <span key={`${id}-${sapSystemSid}`}>
```

Comment:
Had to add a key in the span and in SapSystemLink component to get rid off 
``` Each child in a list should have a unique "key" prop```
Any ideas how to handle it better ?